src/views/onboardingProduct/components/StepAdditionalInformations.tsx

```diff
@@ -14,7 +14,7 @@ import { OnboardingStepActions } from '../../../components/OnboardingStepActions
 import { RadioWithTextField } from '../../../components/RadioWithTextField';
 import { StepperStepComponentProps } from '../../../../types';
 
-export function StepAdditionalInformations({ forward, back }: StepperStepComponentProps) {
+export function StepAdditionalInformations({ forward, back }: StepperStepComponentProps, originId?: string, origin?: string) {
```

Comment:
Create a type called Props that include StepperStepComponentProps and then use origin and originId